code/script.js

```diff
@@ -34,18 +59,164 @@ const showMessage = (message, sender) => {
+  let plantAnswer = inputWrapper.value;
+  showMessage(`I'm a fighter! One calathea for me!`, "user");
+  inputWrapper.value = "";
+  setTimeout(() => question4(plantAnswer), 1000);
+};
+
+const monsteraChoice = (event) => {
+  event.preventDefault(); //prevents refreshing the page
+  let plantAnswer = inputWrapper.value;
+  showMessage(`Holey leaves, give me this Swiss Cheese Plant! Monstera it will be.`, "user");
+  inputWrapper.value = "";
+  setTimeout(() => question4(plantAnswer), 1000);
+};
+
+const begoniaChoice = (event) => {
+  event.preventDefault(); //prevents refreshing the page
+  let plantAnswer = inputWrapper.value;
+  showMessage('I spotted this dotted little flair. Lady Begonia for me, please.', "user");
+  inputWrapper.value = '';
+  setTimeout(() => question4(plantAnswer), 1000);
+};
+
+//Bot asking where to pick up the plant
+const question4 = () => {
+  showMessage("In which store would you like to pick up your plant?", "bot");
+  inputWrapper.innerHTML = `
+  <button id="lillaessingenBtn">Lilla Essingen</button>
+  <button id="sodermalmBtn">Södermalm</button>
+  `
+
+  document.getElementById("lillaessingenBtn").addEventListener("click", () => finalDecision("Lilla Essingen"))
+  document.getElementById("sodermalmBtn").addEventListener("click", () => finalDecision("Sodermalm"))
+  ;}
+
+//Final decision and video 
+const finalDecision = (finalDecision) => {
+
+  if (finalDecision === "Lilla Essingen"){
+    setTimeout(() => inputWrapper.innerHTML = `<span>`, 1000);
+    setTimeout(() => showMessage("Lilla Essingen, I will be there!", "user"), 1500);
+    setTimeout(() => showMessage("Your plant will be available to pick up at Lilla Essingen Funky Plant Store as soon as you see the Philodendron Pink Princess!", "bot"), 1500);
+    setTimeout(() => inputWrapper.innerHTML = `<span>`, 2000);
+    setTimeout(() => showMessage("We're preparing your plant! You will be informed when it's ready! 🧑🏾‍🌾 Please, bear with us a little bit more 💝 ...", "bot"), 1500);
+
+    
+  } 
+  
+  else { 
+    setTimeout(() => inputWrapper.innerHTML = `<span>`, 1000);
+    setTimeout(() => showMessage("Gonna take a walk on Söder, why not to pick it up there...", "user"), 1500);
+    setTimeout(() => showMessage("Your plant will be available to pick up at Södermlam Funky Plant Store as soon as you see the Philodendron Pink Princess!", "bot"), 1500);
+    setTimeout(() => inputWrapper.innerHTML = `<span>`, 2000);
+    setTimeout(() => showMessage("We're preparing your plant! You will be informed when it's ready! 🧑🏾‍🌾 Please, bear with us a little bit more 💝 ...", "bot"), 1500);
+  }
+  
+  setTimeout(() => chat.innerHTML = `
+  <img src="assets/pinklady.jpeg" alt="Thank you image" class="final-image"> <p class="final">Thank you for shopping with us!<br> You have now 24h to pick up your plant at the chosen localisation.</p> 
+  `, 15500);
+  setTimeout(() => inputWrapper.innerHTML = `<span>`, 7000 ); 
```

Comment:
The set time out could probably be cut short a little, the first time I thought it had ended with the text to bare a little bit more, since nothing more happened. Just as I clicked refresh, the image appeared and I realized that there was actually more to come. Perhaps 4-5000 is enough? :-)